tools/slintpad/src/editor_widget.ts

```diff
@@ -592,10 +592,16 @@ export class EditorWidget extends Widget {
             ["", "Hello World!"],
             ["examples/gallery/gallery.slint", "Gallery"],
             ["demos/home-automation/ui/demo-debug.slint", "Home Automation"],
+            ["demos/usecases/ui/app.slint", "Usecases Demo"],
```

Comment:
Use cases are two words :)

```suggestion
            ["demos/usecases/ui/app.slint", "Use Cases Demo"],
```